test/each.spec.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-env mocha */
+const StringHelper = require('../index')
+const expect = require('chai').expect
+const sinon = require('sinon')
+
+describe('#each', () => {
+  let data = [
+    {a: 1, b: 'John'},
+    {a: 2, b: 'Jane'},
+    {a: 3, b: 'Lisa'}
+  ]
+
+  let table = '<table><thead>' +
+  '<tr><th>ID</th><th>Name</th></tr>' +
+  '</thead><tbody>' +
+  '<tr><td>1</td><td>John</td></tr>' +
+  '<tr><td>2</td><td>Jane</td></tr>' +
+  '<tr><td>3</td><td>Lisa</td></tr>' +
+  '</tbody></table>'
+
+  it('sends the correct parameters to the callback', () => {
+    let cbSpy = sinon.spy()
+    const jane = {a: 2, b: 'Jane'}
+    let helper = new StringHelper()
+    let result = helper.each(data, cbSpy).str()
+    expect(result).to.equal('')
+    // console.log(cbSpy.secondCall.args)
```

Comment:
Comments should be used to document code, not to leave dead code that will eventually stink.